algobot/enums.py

```diff
@@ -17,8 +18,31 @@ class GraphType:
 LONG = 1
 SHORT = -1
 
-TRAILING = 2
-STOP = 1
+
+class OrderType:
```

Comment:
Still could be improved but it's a step in the right direction. by improvements we could use real enums and get read of the whole `from_str` but that would require a lot more mapping ;)

Reply:
nice